.gitignore

```diff
@@ -3,7 +3,7 @@
 *.iml
 .env
 .DS_Store
-venv/
+venAnother duty of ORM is to change the database schema: add, delete, and modify tables. As a rule, this is done not on pure SQL but with the help of a special language. This allows you to work with ORM without worrying about the peculiar features of specific databases.v/
```

Comment:
I didn't notice that pycharm added these changes to .gitignore

Reply:
Then revert it)